certified-connectors/Easyvista_Service._Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,13162 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Different endpoints for creating/updating/deleting in the Easyvista Service Manager database / EasyVista Service Manager Modification May 2023",
+    "title": "EasyVista Service Manager",
+    "version": "1.0.0"
```

Comment:
@ezv-compteintegration in the "info" section please add the "contact" property. Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json). 
```
    "contact": {
      "name": "Full name",
      "url": "Contact URL",
      "email": "Contact email address"
    }
```